assets/ui/MainMenu.tscn

```diff
@@ -0,0 +1,81 @@
+[gd_scene load_steps=8 format=2]
```

Comment:
can we move this entire file to an appropriate folder since it's a scene and not an asset?